scoring/src/main/scala/ai/h2o/sparkling/ml/models/H2OMOJOModel.scala

```diff
@@ -89,10 +99,16 @@ abstract class H2OMOJOModel
 
   def getTrainingMetrics(): Map[String, Double] = $(trainingMetrics)
 
+  def getTrainingMetricsObject(): H2OMetrics = $(trainingMetricsObject)
+
   def getValidationMetrics(): Map[String, Double] = $(validationMetrics)
 
+  def getValidationMetricsObject(): H2OMetrics = $(validationMetricsObject)
+
   def getCrossValidationMetrics(): Map[String, Double] = $(crossValidationMetrics)
 
+  def getCrossValidationMetricsObject(): H2OMetrics = $(crossValidationMetricsObject)
```

Comment:
I think this would deserve documentation, explanation these are the combined CV holdout metrics

Reply:
Ok, I will improve that.